src/sync.ts

```diff
@@ -920,15 +903,6 @@ export class SyncApi {
             this.updateSyncState(SyncState.Syncing, syncEventData);
 
             if (this.client.store.wantsSave()) {
-                // We always save the device list (if it's dirty) before saving the sync data:
-                // this means we know the saved device list data is at least as fresh as the
-                // stored sync data which means we don't have to worry that we may have missed
-                // device changes. We can also skip the delay since we're not calling this very
-                // frequently (and we don't really want to delay the sync for it).
-                if (this.syncOpts.crypto) {
-                    await this.syncOpts.crypto.saveDeviceList(0);
-                }
```

Comment:
How does rust crypto handle this? Seems like an important thing to have

Reply:
We don't have it when using EW so I suppose it's fine?

Reply:
Or maybe its the cause of some bugs we haven't yet investigated properly?

Reply:
rust crypto persists the list of users that need a key query to indexeddb on each `/sync` request, so we don't need this extra call to flush out the data.